README.md

```diff
@@ -6,6 +6,7 @@ I'm a webdev student at neue fische.
 ## Lessons Learned
 
 I learned how to build multi-step forms with react and how to process user input. I also learned how to build components in Storybook and how to test them with the react testing library.
+Additionally, I got to know the framer-motion library and used it to implement animations in my app.
```

Comment:
really nice readme 👍 just a typo...I think it´s "I´ve learned" instead of "I learned"